src/lib/y2ntp_client/widgets/main_widgets.rb

```diff
@@ -0,0 +1,263 @@
+require "yast"
+
+require "cwm/widget"
+require "cwm/table"
+
+require "y2ntp_client/dialog/pool"
+
+Yast.import "Confirm"
+Yast.import "NtpClient"
+Yast.import "Popup"
+
+module Y2NtpClient
+  module Widgets
+    # Widget for netconfig policy
+    class PolicyCombo < CWM::ComboBox
+      def initialize
+        textdomain "ntp-client"
+      end
+
+      def label
+        # TRANSLATORS: label for widget that allows to define if ntp configiration is only
+        # from its source or dynamically extended e.g. via DHCP
+        _("Configuration Source")
+      end
+
+      def help
+        # TODO: not written previously, but really deserve something
+        _("<b>Configuration Source</b> defines if network itself can adds ntp sources.")
```

Comment:
```rb
# TRANSLATORS: 'man 8 netconfig' is a command, do not translate that
_("The NTP configuration may be provided by the local network over DHCP. " \
"<b>Configuration Source</b> can simply enable or disable using that configuration. " \
"In cases where there may be multiple DHCP sources, it can prioritize them: " \
"see 'man 8 netconfig'.")
```

Tip: "can adds" is ALWAYS wrong.